.gitignore

```diff
@@ -36,3 +36,8 @@ yarn-debug.log*
 
 # Ignore IDE files
 .idea/
+
+# Environment variables
+.env.development
```

Comment:
@Vladimir-Kuchinskiy maybe we just can use `.env.*`

Reply:
okey